src/main.c

```diff
@@ -513,6 +513,51 @@ static gboolean resign_start(int argc, char **argv)
 	return TRUE;
 }
 
+static gboolean extract_signature_start(int argc, char **argv)
+{
+	RaucBundle *bundle = NULL;
+	GError *ierror = NULL;
+	g_debug("extract signature start");
+
+	if (argc < 3) {
+		g_printerr("An input bundle must be provided\n");
+		r_exit_status = 1;
+		goto out;
+	}
+
+	if (argc < 4) {
+		g_printerr("An output signature file must be provided\n");
+		r_exit_status = 1;
+		goto out;
+	}
+
+	if (argc > 4) {
+		g_printerr("Excess argument: %s\n", argv[4]);
+		r_exit_status = 1;
+		goto out;
+	}
+
+	g_debug("input bundle: %s", argv[2]);
+	g_debug("output file: %s", argv[3]);
+
+	if (!check_bundle(argv[2], &bundle, TRUE, &ierror)) {
```

Comment:
@jluebbe Should it be configurable with `--no-verify` option ?

Reply:
Hmm, probably yes, although I'm not a fan of `--no-verify`.